components/konflux-rbac/policies/validate-rolebindings/.chainsaw-test/chainsaw-test.yaml

```
@@ -0,0 +1,106 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: disallow-tenant-groups
+spec:
+  description: |
+    Asserts that rolebindings to restricted groups are denied in tenant namespaces
+  concurrent: false
+  namespace: 'deny-tenant-groups'
+  steps:
+  - name: "Given namespace has tenant label"
+    try:
+    - apply:
+        file: ./resources/tenant-namespace.yaml
+  - name: "Given cluster policy is ready"
+    try:
+    - apply:
+        file: ../invalid-subjects.yaml
+    - assert:
+        file: ./chainsaw-assert-clusterpolicy.yaml
+  - name: "When invalid rolebinding is applied, assert an error"
+    try:
+    - apply:
+        bindings:
+          group: "system:anonymous"
+        file: ./resources/rolebinding.yaml
```

not sure but we may require

```suggestion
        file: ./resources/rolebinding.yaml
        template: true
```

---

Fixed in b25cb43f.  Thanks!

---

Doesn't work for me; womp womp :(